tests/transform/test_genes_biodomains.py

```diff
@@ -83,7 +83,9 @@ def test_transform_genes_biodomains_should_pass(
 
     @pytest.mark.parametrize("input_file", fail_test_data, ids=fail_test_ids)
     def test_transform_genes_biodomains_should_fail(self, input_file):
-        with pytest.raises(KeyError):
+        with pytest.raises(
```

Comment:
This error type appears to have changed due to the new `pandas` version

Reply:
Odd... I wonder what caused it to change it to the different error type.